internal/runbits/rationalize/types.go

```diff
@@ -17,6 +17,8 @@ var ErrNotAuthenticated = errors.New("not authenticated")
 
 var ErrActionAborted = errors.New("aborted by user")
 
+var ErrHeadless = errors.New("headless")
```

Comment:
I view errors in this file as "global" errors. Is this going to be used outside the internal/runbits/requirements package? If not, we should put it in that package and remove it here.

Reply:
Never mind, I think this will be used in user-facing errors for requirement operations like install/uninstall.